packages/expo-system-ui/ios/ExpoSystemUI/ExpoSystemUIModule.swift

```diff
@@ -9,8 +9,13 @@ public class ExpoSystemUIModule: Module {
     OnCreate {
       // TODO: Maybe read from the app manifest instead of from Info.plist.
       // Set / reset the initial color on reload and app start.
-      let color = Bundle.main.object(forInfoDictionaryKey: "RCTRootViewBackgroundColor") as? Int
-      Self.setBackgroundColorAsync(color: color)
+      let color = UserDefaults.standard.integer(forKey: "backgroundColor")
```

Comment:
Let's use a more expo-specific key here so it doesn't collide with something else.
```suggestion
      let color = UserDefaults.standard.integer(forKey: "ExpoSystemUI.backgroundColor")
```